docs/source/datasets/public_datasets.csv

```diff
@@ -1,20 +1,20 @@
-Dataset,Citation,Samples,Events,Reading Measures
-:py:class:`~pymovements.datasets.BSC`,:cite:p:`BSC`,,X,
-:py:class:`~pymovements.datasets.BSCII`,:cite:p:`BSCII`,,X,
-:py:class:`~pymovements.datasets.CodeComprehension`,:cite:p:`CodeComprehension`,,X,
-:py:class:`~pymovements.datasets.CopCo`,:cite:p:`CopCoL1Hollenstein`,,X,
-:py:class:`~pymovements.datasets.DAEMONS`,:cite:p:`DAEMONS`,,X,
-:py:class:`~pymovements.datasets.DIDEC`,:cite:p:`DIDEC`,,X,
-:py:class:`~pymovements.datasets.EMTeC`,:cite:p:`EMTeC`,,X,
-:py:class:`~pymovements.datasets.FakeNewsPerception`,:cite:p:`FakeNewsPerception`,,X,
-:py:class:`~pymovements.datasets.GazeGraph`,:cite:p:`GazeGraph`,,X,
-:py:class:`~pymovements.datasets.GazeOnFaces`,:cite:p:`GazeOnFaces`,,X,
-:py:class:`~pymovements.datasets.GazeBase`,:cite:p:`GazeBase`,X,,
-:py:class:`~pymovements.datasets.GazeBaseVR`,:cite:p:`GazeBaseVR`,X,,
-:py:class:`~pymovements.datasets.HBN`,:cite:p:`HBN`,X,,
-:py:class:`~pymovements.datasets.InteRead`,:cite:p:`InteRead`,X,,
-:py:class:`~pymovements.datasets.JuDo1000`,:cite:p:`JuDo1000`,X,,
-:py:class:`~pymovements.datasets.PoTeC`,:cite:p:`PoTeC`,X,,
-:py:class:`~pymovements.datasets.Provo`,:cite:p:`Provo`,X,,
-:py:class:`~pymovements.datasets.SBSAT`,:cite:p:`SB-SAT`,X,,
-:py:class:`~pymovements.datasets.UCL`,:cite:p:`UCL`,X,,
+Dataset,Citation,Samples,Events,Reading Measures,# Subjects,# Sessions per Subject,Sampling Rate,Stimulus Type
```

Comment:
Can you also update the Samples, Events and Reading Measures columns? I just copy-pasted them when creating this, so most of them are probably false.